src/widgets/glyph.rs

```diff
@@ -62,15 +68,35 @@ impl Widget for GlyphWidget {
                                  x as f64 + l_pad,
                                  (height +  y) as f64 - baseline,
         ]);
-        let fill_color = if is_active { HIGHLIGHT_COLOR } else { GLYPH_COLOR };
-        let fill = ctx.render_ctx.solid_brush(fill_color).unwrap();
+
+
+        let glyph_body_color = if is_active { HIGHLIGHT_COLOR } else { GLYPH_COLOR };
+        let fill = ctx.render_ctx.solid_brush(glyph_body_color).unwrap();
         ctx.render_ctx.fill(affine * &self.path, &fill, FillRule::NonZero);
+
         if is_hot {
             let outline_color = ctx.render_ctx.solid_brush(HIGHLIGHT_COLOR).unwrap();
             //eprintln!("{} {:?} x{} {:?}", self.glyph.name, bb, scale, rect);
-            ctx.render_ctx.stroke(affine * &self.path, &outline_color, 0.5, None);
-            ctx.render_ctx.stroke(rect, &outline_color, 0.5, None);
+            ctx.render_ctx.stroke(affine * &self.path, &outline_color, 1.0, None);
+            ctx.render_ctx.stroke(rect, &outline_color, 1.0, None);
         }
+
+        // draw the glyph name:
+        let font_size = 12.0;
+        let name_color = if is_hot { HIGHLIGHT_COLOR } else { GLYPH_COLOR };
+        let text = get_text_layout(&mut ctx.render_ctx, &self.glyph.name, font_size);
+        let xpos = geom.pos.0 + (geom.size.0 - text.width() as f32) / 2.;
```

Comment:
I tend to prefer * 0.5 because it's more efficient but the actual impact is negligible (ie I use performance as a guidepost to be consistent).